finetune/saver.py

```diff
@@ -260,74 +227,24 @@ def init_fn(scaffold, session, model_portion=None, refresh_base_model=False):
                 variables_sv = dict()
             all_vars = tf.global_variables()
 
-            zero_out_adapters = False
-            if (
-                model_portion != "entire_model"
-            ):  # we must be loading in the case of two separate estimators
-                all_vars, zero_out_adapters = self.subset_to_load(
-                    model_portion, refresh_base_model, all_vars
-                )
-
             global_step_var = tf.train.get_global_step()
 
             for var in all_vars:
-                if global_step_var is not None and global_step_var.name == var.name:
-                    continue
+#                if global_step_var is not None and global_step_var.name == var.name:
```

Comment:
What's the verdict on this bit?

Reply:
Oh, this was an accident. I do find myself commenting this out regularly to restart stuff and rarely needing it to be there though. Could instead hide it behind a config argument?